docs/values.schema.json

```diff
@@ -48,9 +48,91 @@
     "global": {
       "description": "Global variables",
       "type": "object",
-      "additionalProperties": true
+      "additionalProperties": true,
+      "properties": {
+        "repositoryName": {
+          "description": "Application name, default to GIT repository name",
+          "type": "string"
+        },
+        "namespace": {
+          "description": "Default namespace",
+          "type": "string"
+        },
+        "registry": {
+          "description": "Default docker registry URL",
```

Comment:
```suggestion
          "description": "Default docker registry URL, it is recommended tu use .kontinuous/config.yaml file to setup registry instead of using values",
```